diff -up Fabric-0.9.2/setup.py.orig Fabric-0.9.2/setup.py
--- Fabric-0.9.2/setup.py.orig	2010-10-09 09:41:57.769429322 -0400
+++ Fabric-0.9.2/setup.py	2010-10-09 09:42:37.986679839 -0400
@@ -31,7 +31,7 @@ setup(
     packages=find_packages(),
     test_suite='nose.collector',
     tests_require=['nose', 'fudge'],
-    install_requires=['pycrypto <2.1', 'paramiko >=1.7.6'],
+    install_requires=['pycrypto >=1.9', 'paramiko >=1.7.6'],
     entry_points={
         'console_scripts': [
             'fab = fabric.main:main',
