diff -up lxc-0.6.3/configure.ac.orig lxc-0.6.3/configure.ac
--- lxc-0.6.3/configure.ac.orig	2009-07-28 00:59:14.737542987 -0400
+++ lxc-0.6.3/configure.ac	2009-07-28 01:03:22.907540449 -0400
@@ -25,10 +25,11 @@ AS_AC_EXPAND(LOCALSTATEDIR, $localstated
 AS_AC_EXPAND(LXCPATH, "${localstatedir}/lib/lxc")
 AS_AC_EXPAND(LXC_GENERATE_DATE, "$(date)")
 
-AC_CHECK_HEADERS([linux/netlink.h linux/genetlink.h], [], AC_MSG_ERROR([netlink headers not found. Please install the linux kernel headers.]),
-[#include <linux/types.h>
-#include <bits/sockaddr.h>
-#include <linux/socket.h>])
+AC_CHECK_HEADERS([linux/netlink.h linux/genetlink.h],
+                 [],
+                 AC_MSG_ERROR([netlink headers not found. Please install the linux kernel headers.]),
+                 [#include <sys/socket.h>
+    ])
 
 AC_CHECK_HEADERS([sys/capability.h], [], AC_MSG_ERROR([please install libcap-devel.]),
 [#include <sys/types.h>
