diff -up scribe/configure.ac.orig scribe/configure.ac
--- scribe/configure.ac.orig	2009-05-05 16:27:47.000000000 -0600
+++ scribe/configure.ac	2009-05-05 16:28:38.000000000 -0600
@@ -57,8 +57,8 @@ FB_WITH_PATH([fb303_home], [fb303path], 
 FB_WITH_PATH([smc_home], [smcpath], [${EXTERNAL_PATH}/services/trunk/src])
 FB_WITH_PATH([fb_home], [fbpath], [${EXTERNAL_PATH}/libfacebook])
 
-# Require boost 1.36 with system and filesytem libraries
-AX_BOOST_BASE([1.36])
+# Require boost 1.33 with system and filesytem libraries
+AX_BOOST_BASE([1.33])
 AX_BOOST_SYSTEM
 AX_BOOST_FILESYSTEM
 
diff -up scribe/src/file.cpp.orig scribe/src/file.cpp
--- scribe/src/file.cpp.orig	2009-05-05 16:26:56.000000000 -0600
+++ scribe/src/file.cpp	2009-05-05 16:27:16.000000000 -0600
@@ -209,7 +209,7 @@ void StdFile::listImpl(const std::string
       boost::filesystem::directory_iterator dir_iter(path), end_iter;
 
       for ( ; dir_iter != end_iter; ++dir_iter) {
-        _return.push_back(dir_iter->filename());
+        _return.push_back(dir_iter->leaf());
       }
     }
   } catch (std::exception const& e) {
