diff -up mongodb-mongo-e316c78/SConstruct.orig mongodb-mongo-e316c78/SConstruct
--- mongodb-mongo-e316c78/SConstruct.orig	2009-12-06 16:56:43.186678071 -0500
+++ mongodb-mongo-e316c78/SConstruct	2009-12-06 18:32:10.894675691 -0500
@@ -1204,8 +1204,6 @@ def installBinary( e , name ):
     fullInstallName = installDir + "/bin/" + name
 
     allBinaries += [ name ]
-    if solaris or linux:
-        e.AddPostAction( inst, e.Action( 'strip ' + fullInstallName ) )
 
     if linux and len( COMMAND_LINE_TARGETS ) == 1 and str( COMMAND_LINE_TARGETS[0] ) == "s3dist":
         e.AddPostAction( inst , checkGlibc )
