diff -up redis-1.2.6/redis.conf.copy.orig redis-1.2.6/redis.conf.copy
--- redis-1.2.6/redis.conf.copy.orig	2010-08-16 23:21:17.139537736 -0400
+++ redis-1.2.6/redis.conf.copy	2010-08-16 23:21:05.476662476 -0400
@@ -2,19 +2,18 @@
 
 # By default Redis does not run as a daemon. Use 'yes' if you need it.
 # Note that Redis will write a pid file in /var/run/redis.pid when daemonized.
-daemonize no
+daemonize yes
 
 # When run as a daemon, Redis write a pid file in /var/run/redis.pid by default.
 # You can specify a custom pid file location here.
-pidfile /var/run/redis.pid
+pidfile /var/run/redis/redis.pid
 
 # Accept connections on the specified port, default is 6379
 port 6379
 
 # If you want you can bind a single interface, if the bind option is not
 # specified all the interfaces will listen for connections.
-#
-# bind 127.0.0.1
+bind 127.0.0.1
 
 # Close the connection after a client is idle for N seconds (0 to disable)
 timeout 300
@@ -24,12 +23,12 @@ timeout 300
 # debug (a lot of information, useful for development/testing)
 # notice (moderately verbose, what you want in production probably)
 # warning (only very important / critical messages are logged)
-loglevel debug
+loglevel notice
 
 # Specify the log file name. Also 'stdout' can be used to force
 # the demon to log on the standard output. Note that if you use standard
 # output for logging but daemonize, logs will be sent to /dev/null
-logfile stdout
+logfile /var/log/redis/redis.log
 
 # Set the number of databases. The default database is DB 0, you can select
 # a different one on a per-connection basis using SELECT <dbid> where
@@ -64,7 +63,7 @@ dbfilename dump.rdb
 
 # For default save/load DB in/from the working directory
 # Note that you must specify a directory not a file name.
-dir ./
+dir /var/lib/redis/
 
 ################################# REPLICATION #################################
 
diff -up redis-1.2.6/redis.copy.orig redis-1.2.6/redis.copy
